controllers/apply/form-router-next/views/eligibility.njk

```diff
@@ -34,6 +27,13 @@
                     </p>
                 </div>
             {% else %}
+                {% set prefix = __('global.misc.stepProgress', currentStepNumber, totalSteps) %}
```

Comment:
Realised whilst testing this that this header doesn't make sense on the success / ineligible screens to switched this around